.codeclimate.yml

```diff
@@ -0,0 +1,15 @@
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        javascript:
+          mass_threshold: 100
```

Comment:
Yeah, I was trying to see what effect it would have. The default is 40. Higher produces fewer warnings.

That definition is pretty hard for me to intuitively apply to code, so I'm not sure how to set that value other than trial and error.